charts/stable/mealie/values.yaml

```diff
@@ -1,34 +1,28 @@
-#
 # IMPORTANT NOTE
 #
 # This chart inherits from our common library chart. You can check the default values/options here:
 # https://github.com/k8s-at-home/library-charts/tree/main/charts/stable/common/values.yaml
 #
-
+---
 image:
   # -- image repository
   repository: hkotel/mealie
   # -- image tag
   # @default -- chart.appVersion
-  tag:
+  tag: frontend-v1.0.0beta-2
   # -- image pull policy
   pullPolicy: IfNotPresent
 
-# -- environment variables. See [image docs](https://hay-kot.github.io/mealie/documentation/getting-started/install/#env-variables) for more details.
-# @default -- See below
-env:
-  # -- Set the container timezone
-  TZ: UTC
-  # -- Set the application database type
-  DB_ENGINE: sqlite
-
```

Comment:
There still seem to be quite some env vars according to the docs. Please don't completely remove this section. Perhaps only remove the `DB_ENGINE` var?